ios/ItaliaApp/it.lproj/InfoPlist.strings

```diff
@@ -1,6 +1,6 @@
-"NSCameraUsageDescription" = "IO ha necessità di accedere alla fotocamera per la scansione dei codici di pagamento";
+"NSCameraUsageDescription" = "Potrai leggere codici QR e pagare gli avvisi pagoPA.";
```

Comment:
I set my language device to english and I get a message that is not sync with the italian one
<img src="https://user-images.githubusercontent.com/822471/133741631-0505f3c6-f8ce-4592-a220-987db3da62c0.jpg" height="600"/>
Perhaps should we update these too?
https://github.com/pagopa/io-app/blob/9fbe14ea84b2e3246c92b32a3eeff899c8a0622c/ios/ItaliaApp/Info.plist#L62

Reply:
Yes, changed in 7c67d70